examples/teams/variables.tf

```diff
@@ -2,8 +2,3 @@ variable "github_token" {
   description = "Token used to apply configuration changes"
   type        = string
 }
-
-variable "json_file" {
```

Comment:
why was this variable removed from required? dont we need json to at least get the team name

Reply:
This is in the example folder. we don't need it